test/edgecases.jl

```diff
@@ -0,0 +1,25 @@
+using Test, Distributions, StatsBase
+
+@testset "TDist(Inf) is Normal(0,1)" begin
+    T = TDist(Inf)
+    N = Normal(0, 1)
+
+    x = rand(N)
+    z = rand(T, 10000000)
+
+    @test mean(T) == mean(N)
+    @test median(T) == median(N)
+    @test mode(T) == mode(N)
+    @test var(T) == var(N)
+    @test skewness(T) == skewness(N)
+    @test kurtosis(T) == kurtosis(N)
+    #@test entropy(T) == entropy(N)
+    @test pdf(T, x) ≈ pdf(N, x)
+    @test logpdf(T, x) ≈ logpdf(N, x)
+    #@test gradlogpdf(T, x) ≈ gradlogpdf(N, x)
```

Comment:
why the commented tests?

Reply:
`entropy`, `gradlogpdf`, and `cf` are currently not rigged to give the same output as for `Normal(0,1)`, so we have:
```julia
using Distributions
d = TDist(Inf)

julia> entropy(d)
NaN

julia> gradlogpdf(d, 1.0)
NaN

julia> cf(d, 0.25)
ERROR: AmosException with id 4: input argument magnitude too large, complete loss of accuracy by argument reduction.
Stacktrace:
 [1] _besselk(::Float64, ::Complex{Float64}, ::Int32) at /Users/johnczito/.julia/packages/SpecialFunctions/fvheQ/src/bessel.jl:239
 [2] besselk at /Users/johnczito/.julia/packages/SpecialFunctions/fvheQ/src/bessel.jl:354 [inlined]
 [3] besselk at /Users/johnczito/.julia/packages/SpecialFunctions/fvheQ/src/bessel.jl:450 [inlined]
 [4] cf(::TDist{Float64}, ::Float64) at /Users/johnczito/.julia/packages/Distributions/Iltex/src/univariate/continuous/tdist.jl:82
 [5] top-level scope at none:0
```
Those tests would currently fail if they were included.

Reply:
ok, fine for this behaviour